CONTRIBUTING.md

```diff
@@ -97,16 +97,14 @@ docker compose exec -it client /bin/sh
 To start the local Firezone cluster, follow these steps:
 
 ```
-docker compose build
-docker compose up -d postgres
 docker compose run --rm elixir /bin/sh -c "cd apps/domain && mix ecto.create && mix ecto.migrate && mix ecto.seed"
 
 # Before moving to the next step, copy the Firezone account UUID from the seed step
 # Here's an example of the output
     Created accounts:
     c89bcc8c-9392-4dae-a40d-888aef6d28e0: Firezone Account
 
-docker compose up -d api web vault gateway client relay
+docker compose up -d
```

Comment:
The dependencies are already in docker-compose.yml, I haven't needed to explicitly up the containers yet